k8s/pkg/apis/longhorn/v1beta2/volume.go

```diff
@@ -224,9 +224,12 @@ type VolumeSpec struct {
 	NodeID string `json:"nodeID"`
 	// +optional
 	MigrationNodeID string `json:"migrationNodeID"`
+	// Deprecated: Replaced by field `image`.
```

Comment:
Create an issue to remove this deprecated field in 1.7, the next feature release.

Reply:
Tracked in https://github.com/longhorn/longhorn/issues/6684